docs/6-kartlegging.adoc

```diff
@@ -31,9 +31,9 @@ Eksempler på avgrensing av et datasett:
 * Vernepliktige er etter lov om verneplikt en del av Forsvarets verneplikts- og tjenesteregister.
 
 
-Det kan oppstå behov for å definere begrep i kartleggingsfasen og i et utviklingsprosjekt, og vi anbefaler at disse forvaltes i en felles begrepskatalog for virksomheten med tydelige forvaltningsprosesser definert (mer om dette i kapittelet om Styring og forvaltning). Begrepene bør være definert i henhold til angitt https://www.difi.no/fagomrader-og-tjenester/digitalisering-og-samordning/standarder/standarder/standard-begrepsbeskrivelser[standard]. Målet på sikt er at etatenes begreper har en https://www.difi.no/fagomrader-og-tjenester/digitalisering-og-samordning/standarder/referansekatalogen/pekere-til-offentlige-ressurser-pa-nett[unik identifikator] som gjør at begrepet er en felles ressurs som kan pekes til og gjenbrukes på tvers av offentlig forvaltning. F.eks  fra både intern og felles datakatalog og i definerte informasjonsmodeller.
+Det kan oppstå behov for å definere begrep i kartleggingsfasen og i et utviklingsprosjekt, og vi anbefaler at disse forvaltes i en felles begrepskatalog for virksomheten med tydelige forvaltningsprosesser definert (mer om dette i kapittelet om Styring og forvaltning). Begrepene bør være definert i henhold til angitt https://www.digdir.no/digitalisering-og-samordning/omgrepsanalyse-og-definisjonsarbeid/1483[standard]. Målet på sikt er at etatenes begreper har en https://www.digdir.no/digitale-felleslosninger/pekere-til-offentlige-ressurser-pa-nett/1492[unik identifikator] som gjør at begrepet er en felles ressurs som kan pekes til og gjenbrukes på tvers av offentlig forvaltning. F.eks  fra både virksomhetsintern og felles datakatalog og i definerte informasjonsmodeller.
```

Comment:
"F.eks" -> "F.eks." (iht. Språkrådet skal det være punktum bak denne forkortelsen, https://www.sprakradet.no/sprakhjelp/Skriveregler/Forkortinger/#f) - kan hende at det er flere forkortelse i veilederen som også bør rettes opp, har ikke sjekket.

Reply:
Jeg foreslår litt kortere setninger. Da kan det bli enklere for å lesere å forstå:

Det kan oppstå behov for å definere begreper. Vi anbefaler at begrepene forvaltes i en felles intern begrepskatalog og at det defineres en tydelig prosess for arbeidet med å definere begreper (mer om dette i kapittelet om Styring og forvaltning). Begrepene bør være definert i henhold til angitt https://www.digdir.no/digitalisering-og-samordning/omgrepsanalyse-og-definisjonsarbeid/1483[standard]. Målet er at etatenes begreper har en https://www.digdir.no/digitale-felleslosninger/pekere-til-offentlige-ressurser-pa-nett/1492[unik identifikator] som gjør at begrepet blir en felles ressurs som andre kan peke til. Da kan begrepsdefinisjoner gjenbrukes på tvers av offentlig forvaltning, for eksempel fra virksomhetsinterne datakataloger, fra Felles datakatalog og i definerte informasjonsmodeller.